### PR TITLE
Implement div operation

### PR DESCRIPTION
**Pull request by JMJustas**
This PR implements a division operation

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -6,6 +6,10 @@ function sub(a, b) {
     return a - b;
 }
 
+function div(a, b) {
+    return a / b;
```
Comment (domaspe) [inline main.js]: Throw an error if operand `b` is 0

**Review by domaspe — CHANGES_REQUESTED** [review]
Good job.

One comment below

File: main.js (hunk 1 above)
Comment (JMJustas) [reply main.js]: will do

**Comment by domaspe** [comment]
Next time please add a description :+1:

**Pull request merged**